mono/metadata/domain.c

```diff
@@ -786,7 +786,6 @@ mono_init_internal (const char *filename, const char *exe_filename, const char *
 	mono_defaults.generic_ienumerator_class = mono_class_load_from_name (
 	        mono_defaults.corlib, "System.Collections.Generic", "IEnumerator`1");
 
-#ifndef ENABLE_NETCORE
 	ERROR_DECL (error);
```

Comment:
had to enable it because runtime crashed somewhere trying to access unitialized threadpool_perform_wait_callback_method (and also added `_ThreadPoolWaitCallback` for that)